kaitlyn-mcgoldrick/my-app/src/Components/form.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {value: ''};
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange = (event) => {
+        this.setState({value: event.target.value})
+    };
+
+    handleSubmit = (event) => {
+        this.props.handleFormSubmit(this.state.value);
```

Comment:
Nice! I like this abstraction of the form component that takes a custom submit handler from its parent (in this case Pokemon app)! This is cool because you could reuse it in a different project if you wanted!